umd/README.md

```diff
@@ -0,0 +1,22 @@
+**Fetch dependencies:**
+```
+npm install
+```
+
+**Build and run sample in browser (using port 8080)**
+```
+npm run browser

```

Comment:
i would say it is more useful to have the commands listed out. most of the time ppl are looking at these to know how to setup things, hiding the calls to tsc would not be that helpful to them.